### PR TITLE
Lf header 239

### PR DESCRIPTION
**Pull request by LucaFumis**
This PR adds an optional main hamburger menu to the NavBar component.
This is used in the platform rewrite and can be configured.

### CODE REVIEW

File: lib/components/HeaderMenu.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+          transition
+          disablePortal
+          placement={placement || 'bottom-start'}
+        >
+          {({ TransitionProps }) => (
+            <Fade {...TransitionProps}>
+              <Paper>
+                <ClickAwayListener onClickAway={this.handleMenuClose}>
+                  <MenuList
+                    id="menu-list-grow"
```
Comment (mirandaio) [inline lib/components/HeaderMenu.js]: Looks like this `id` is not being referenced anywhere. Could we remove it?

Comment (LucaFumis) [reply lib/components/HeaderMenu.js]: Removed redundant id - all done.

**Comment by LucaFumis** [comment]
Updated Link component with optional className classes in place of inline style as before.

**Comment by LucaFumis** [comment]
Thanks for the feedback @mirandaio - it's all been addressed

**Pull request merged**